go.mod

```diff
@@ -34,6 +34,7 @@ require (
 	github.com/lib/pq v1.10.4
 	github.com/markbates/pkger v0.17.1
 	github.com/mitchellh/mapstructure v1.4.3
+	github.com/onsi/ginkgo v1.16.4
```

Comment:
This one needs to be removed. 
This is causing the:

panic: /tmp/go-build1444516780/b1195/state.test flag redefined: ginkgo.seed